### DISCUSSION TITLE
Triggering Custom Javascript on Feature Click in Choropleth Maps

### DISCUSSION THREAD

**Discussion by jereze** [neocarto/bertin#119] [Q&A] [answered]
Hello Bertin community,

I'm working on creating my inaugural choropleth map with bertin. I was wondering, how can I call a custom JavaScript function when a user interacts with a feature (specifically, a country in my context)? Ideally, I'd like this function to access the complete feature object, or at a minimum, its properties.

I appreciate any guidance or examples you can provide.

Thank you!

**Comment by neocarto** [comment]
It's possible within Observable notebooks. See https://observablehq.com/@neocartocnrs/lets-play-with-viewof?collection=@neocartocnrs/bertin

**Reply by jereze** [reply, marked as answer]
Thank you for your response.

I don't operate within Observable notebooks, but your insight provided a hint on how to achieve my goal.

Here's the solution I came up with:

```js
document.getElementById('bertinmap').addEventListener('click', (event) => {
    console.log(event.target.__data__.properties)
});
```
